### ISSUE TITLE
Add common options to the LinkTo selection in image properties

### ISSUE THREAD

**Issue by SergioEstevao** [wordpress-mobile/WordPress-Editor-iOS#600]
From an user:

> One thing my user also mentioned is that they are also not able to easily set the image to link to the media URL (in the app) after adding it to their post. When editing the image, you can easily edit the Link To: field, but there is no way to pre-fill it with the media URL, and I can’t imagine trying to type it out by hand on a mobile device.
>
> Can we add a selector for Link To that has a few of the common options (Media File, Custom URL, None) that you see in the desktop editor?

@thianhlu any thoughts around this?

**Comment by thianhlu** [comment]
@SergioEstevao good idea! We could add a selector like the `alignment` option or just prefill with the media file url and have it preselected upon tap so they can quickly erase or write over with their custom url

**Comment by SergioEstevao** [comment]
I've implement #599 so now the link info will be pre-filed now, thanks @aerych !

**Comment by meremagee** [comment]
also logged in main ios repo: https://github.com/wordpress-mobile/WordPress-iOS/issues/3366
